### PR TITLE
Asset Hubs: auto incremented asset id for trust backed assets

### PR DESCRIPTION
**Pull request by muharem**
Setup auto incremented asset id to `50_000_000` for trust backed assets.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -1080,6 +1059,11 @@ pub type Migrations = (
 	frame_support::migrations::RemovePallet<DmpQueueName, RocksDbWeight>,
 	pallet_collator_selection::migration::v2::MigrationToV2<Runtime>,
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
+	pallet_assets::migration::next_asset_id::SetNextAssetId<
```
Comment (bkchr) [inline system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:1062]: @joepetrowski still needs to delete `MAX_PRIME` :P

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
@anaelleltd this needs to be communicated to asset issuers.

**Comment by anaelleltd** [comment]
> @anaelleltd this needs to be communicated to asset issuers.

As indicated in **step 6** of the guidelines re: [Runtime release process](https://docs.google.com/document/d/1jWLPI9JIsr5DobyXblrL6GCwGPe5N0lSmClj5MwMsgc/edit?usp=sharing), the **author** of the PR should indicate:
- what kind of changes the PR introduces (breaking? disruption? downtime?)
- who will be affected by theses changes in practice (parachains? wallets? UIs? CEXes? DEXes?)
- how the team/builders affected need to follow-up (recommended changes/updates to the code base)

cc @muharem

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs (hunk 1 above)
Comment (joepetrowski) [reply system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:1062]: Done :)
https://assethub-kusama.subscan.io/extrinsic/7559849-2

**Comment by muharem** [comment]
I have update a description with more information for clients.

@bkchr @joepetrowski should we include this into 1.3?

**Comment by joepetrowski** [comment]
Yeah it will be included from master.

**Comment by muharem** [comment]
@kianenigma you mean as I dynamic parameter? I think no, we do not need it to be a dynamic paramater

**Comment by kianenigma** [comment]
> @kianenigma you mean as I dynamic parameter? I think no, we do not need it to be a dynamic paramater

No, I mean a runtime API to get the next available AssetId, abstracting over where this storage live, and what its semantics are.

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs (hunk 1 above)
Comment (Salimtaura00) [reply system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:1061]: j6A5^$wadK$8);n

**Pull request merged**